### PR TITLE
Ci love

### PR DESCRIPTION
**Pull request by bendavies**
Some clean up to hopefully stop random test failures

### CODE REVIEW

**Comment by bendavies** [comment]
@henrikbjorn this is good now. rebased on master

File: tests/ConsumerTest.php [hunk 1]
```diff
@@ -57,7 +57,7 @@ public function testEmitsConsumeEvent()
         $this->dispatcher->expects($this->at(2))->method('dispatch')
             ->with('bernard.acknowledge', new EnvelopeEvent($envelope, $queue));
 
-        $this->assertTrue($this->consumer->tick($queue));
+        self::assertTrue($this->consumer->tick($queue));
```
Comment (henrikbjorn) [inline tests/ConsumerTest.php]: why this change?

**Comment by bendavies** [comment]
@henrikbjorn are you happy with it?

File: tests/ConsumerTest.php (hunk 1 above)
Comment (mente) [inline tests/ConsumerTest.php]: Sebastian Bergmann admits it was a mistake to make `assert*` methods static. Why would you enforce it like this? IMHO makes sense to at at least split it to 2 PRs. `assert*` update is opinionated while the rest is no-brainer.

**Comment by henrikbjorn** [comment]
I would really like the assertions not to be called statically.

**Comment by bendavies** [comment]
@henrikbjorn done

**Comment by bendavies** [comment]
thanks!
